variables.tf

```
@@ -42,6 +42,106 @@ variable "helm_additional_values" {
   default     = []
 }
 
+variable "node_affinity" {
+  description = "Node affinity settings for Kyverno pods. Use weight as map key; operator is `In` and policy is `preferredDuringSchedulingIgnoredDuringExecution`."
+  type = map(object({
+    key    = string
+    values = list(string)
+  }))
+  default = null
+}
+
+variable "admissioncontroller_node_affinity" {
+  description = "Node affinity settings for admissioncontroller pods. Use weight as map key; operator is `In` and policy is `preferredDuringSchedulingIgnoredDuringExecution`."
```

```suggestion
  description = "Node affinity settings for `admissioncontroller` deployment. Use weight as map key; operator is `In` and policy is `preferredDuringSchedulingIgnoredDuringExecution`. If not null, the `node_affinity` value will be completely overwritten for the `admissioncontroller` deployment."
```

Use the same description for each component.

Please change "pods" with "deployment" in each description.